### PR TITLE
chore: upgrade dependencies & update npm scripts

### LINKED ISSUE #63: fastify not found, proceeding anyway
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

latest

### Plugin version

latest

### Node.js version

latest

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

latest

### Description

Trying to import fastify-flash with
```import flash from 'fastify-flash';```
but getting notification. Is this plugin compatible with ESM?

### Steps to Reproduce

Try to import with ```import flash from 'fastify-flash';```

### Expected Behavior

_No response_

### PR DESCRIPTION
**Pull request by darkgl0w**
Hello.

This PR aims to :
- upgrade package dependencies
- cleanup and update `package.json` scripts

Should fix https://github.com/fastify/fastify-flash/issues/63

#### Checklist

- [x] run `npm run test` ~~and `npm run benchmark`~~
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by darkgl0w** [comment]
@mcollina :smile: yes I will rebase. ^^
What do you prefer about the pretest script ?
`"pretest": "./node_modules/.bin/fastify-secure-session > secret-key"` or should I keep `"pretest": "npx fastify-secure-session > secret-key"` so we can run the tests in windows and mac environments too ?

**Comment by mcollina** [comment]
npx is better

**Comment by darkgl0w** [comment]
I will restore mac and windows runners in CI config too and it will good to go ^^

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -12,8 +12,8 @@
     "lint": "tslint --fix -c tslint.json -t stylish -p ./tsconfig.json",
     "update": "npx npm-check -u",
     "prepublishOnly": "npm run build",
-    "pretest": "./node_modules/.bin/fastify-secure-session > secret-key",
-    "test": "tap tests/**/*.ts",
+    "pretest": "npx fastify-secure-session > secret-key",
+    "test": "tap \"tests/**/*.ts\"",
```
[the remark this answers is not in the document]
Comment (darkgl0w) [reply package.json]: Done and it feels cleaner.
We should do this for the other repositories :stuck_out_tongue:

**Review by jsumners — APPROVED** [review]

**Pull request merged**